### PR TITLE
Omsmith/org image use link

### PR DESCRIPTION
**Pull request by omsmith**

### CODE REVIEW

File: src/es6/Entity.js [hunk 1]
```diff
@@ -103,27 +103,38 @@ export class Entity extends EntitySirenProperties {
...
 		// Clean up if that href has already been added.
-		if (this._subEntities.has(href)) {
-			dispose(this._subEntities.get(href));
+		if (this._subEntities.has(link.href)) {
+			dispose(this._subEntities.get(link.href));
 		}
 		this._subEntitiesLoadStatus.push(new Promise((resolve) => {
-			entityFactory(entityType, href, this._token, (entity) => {
-				this._subEntities.set(href, entity);
+			entityFactory(entityType, link, this._token, (entity) => {
```
Comment (omsmith) [inline src/es6/Entity.js]: Support added in d2l-polymer-siren-behaviors@1.4.0

**Comment by omsmith** [comment]
Have tests running locally now, will sort through what updates need to happen there.

File: src/es6/Entity.js (hunk 1 above)
Comment (jopnick) [inline src/es6/Entity.js]: shouldn't this be `link.href`?

Comment (omsmith) [reply src/es6/Entity.js]: Nope. It's passing the Link object to polymer-siren-behaviors so they the `rel` on it can be inspected

**Comment by omsmith** [comment]
Had to delete Travis' cache to get tests to pass due to the polymer-siren-behaviors version bump (because it isn't an explicit bump here)

**Pull request merged**